aip/general/0136.md

```diff
@@ -119,6 +116,23 @@ An exception to this is for rarely-used, fundamentally imperative operations,
 such as a `Move` or `Rename` operation, for which there would not be an
 expectation of declarative support.
 
+## Rationale
+
+### HTTP path
+
+Similar to standard methods, a custom method should have a `name` or `parent`
```

Comment:
Can this be worded to sound like a statement of fact rather than guidance? e.g.

> Similar to standard methods, a custom method needs a `name` or `parent` field to indicate the resource that...

Reply:
done.